odl/tomo/backends/astra_cpu.py

```diff
@@ -191,6 +191,17 @@ def astra_cpu_backward_projector_call(proj_data, geometry, reco_space,
     # Run algorithm and delete it
     astra.algorithm.run(algo_id)
 
+    # Fix inconsistent scaling
+    # angle interval weight by approximate cell volume
```

Comment:
Depends on what we mean by the projection, if it is the radon transform, this should be independent of the interval length?